observelib/observe/src/main/java/module-info.java

```diff
@@ -3,4 +3,5 @@
     requires io.ballerina.runtime;
     requires opentracing.api;
     requires io.ballerina.config;
+    requires slf4j.api;
```

Comment:
Do we need this ?

Reply:
This for logging the span not found error